### DISCUSSION TITLE
Accepting PRs for openapi-typescript 5.x?

### DISCUSSION THREAD

**Discussion by mitchell-merry** [openapi-ts/openapi-typescript#1505] [General]
Hi @drwpow !

I am wondering if PRs are accepted for 5.x of `openapi-typescript`. We want to avoid forking if possible, and we have a specific requirement for this version (we are still on swagger 2, *groan*).

It has to do with this issue: https://github.com/drwpow/openapi-typescript/issues/1083. I want to add support for `x-nullable` generating `| null` in types. There's a suggestion in that issue to use the `formatter` option, but since the `formatter` passes in the full schema object (and not the type that `openapi-typescript` generates by default), I think it would require re-implementing that logic. I tried calling out to internal functions to get this done... and couldn't make it work.

If I were to raise a PR to support this, behind a CLI flag, would it be accepted and released?

Thanks!

**Comment by drwpow** [comment]
Sure! As long as it doesn’t break existing behavior, I’d be happy to accept any additions to previous versions.
